app/controllers/records_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class RecordsController < ApplicationController
+  include Records::Filters
+
+  before_action :authorize, :not_guest, :not_author, :not_owner, :not_manager
+
+  respond_to :html, :json
```

Comment:
Esto mejor arriba del before_action.
```suggestion
  respond_to :html
```

Reply:
dale lo dejo asi.